Sources/TensorFlow/Operators/NN.swift

```diff
@@ -434,6 +529,31 @@ public func conv3D<Scalar: TensorFlowFloatingPoint>(
         padding: padding.raw)
 }
 
+/// Computes a 2-D Depthwise convolution using `self` as input, with the specified
```

Comment:
- The word "depthwise" is not a proper noun that should be capitalized. Use `depthwise` instead.
- There's no `self` anymore after #138. It appears that #138 didn't update comments to remove mentions of `` `self` `` in other convolution and pooling APIs. We should file an issue and fix them.

```suggestion
/// Computes a 2-D depthwise convolution with the specified input, filter, ...
```